lib/glific/profiles.ex

```diff
@@ -204,8 +207,16 @@ defmodule Glific.Profiles do
     }
 
     case create_profile(attrs) do
-      {:ok, _profile} ->
-        {context, Messages.create_temp_message(context.organization_id, "Success")}
+      {:ok, profile} ->
+        indexed_profile = get_indexed_profile(context.contact)
+
+        {_profile, profile_index} =
+          Enum.find(indexed_profile, fn {index_profile, _index} ->
+            index_profile.id == profile.id
+          end)
+
+        action = Map.put(action, :value, to_string(profile_index))
+        handle_flow_action(context, action, "Switch Profile")
```

Comment:
Can we make Switch profile a tuple for the main function?

Reply:
@pankaj-ag Can you elaborate a bit. Not sure, what changes you are requesting here

Reply:
'''

handle_flow_action(context, action, :switch_profile)